src/moderatorFrontendMain/kotlin/util/Strings.kt

```diff
@@ -7,600 +7,610 @@ import com.studo.campusqr.common.utils.LocalizedString
-    "Zugangsverwalter können Zugangsberechtigungen zu bestehenden Orten einsehen, vergeben und bearbeiten, um eine Zugangskontrolle an der Hochschule zu ermöglichen."
+      "Access managers can view, assign and edit access permissions to existing locations to enable a university access control system.",
+      "Zugangsverwalter können Zugangsberechtigungen zu bestehenden Orten einsehen, vergeben und bearbeiten, um eine Zugangskontrolle an der Hochschule zu ermöglichen."
   )
 
   val user_administration_hint2 = LocalizedString(
-    "Moderators can additionally list and edit locations, download check-in data and report infections.",
-    "Moderatoren können zusätzlich Orte auflisten und bearbeiten, Check-In-Daten herunterladen und Infektionen melden."
+      "Moderators can additionally list and edit locations, download check-in data and report infections.",
+      "Moderatoren können zusätzlich Orte auflisten und bearbeiten, Check-In-Daten herunterladen und Infektionen melden."
   )
 
   val user_administration_hint3 = LocalizedString(
-    "Administrators can additionally create, delete and edit users (access managers, moderators and other administrators).",
-    "Administratoren können zusätzlich Benutzer (Zugangsveralter, Moderatoren und weitere Administratoren) erstellen, löschen und bearbeiten."
+      "Administrators can additionally create, delete and edit users (access managers, moderators and other administrators).",
+      "Administratoren können zusätzlich Benutzer (Zugangsveralter, Moderatoren und weitere Administratoren) erstellen, löschen und bearbeiten."
   )
 
   val user_administration_external_auth_provider = LocalizedString(
-    "Users managed by LDAP.",
-    "User werden durch LDAP verwaltet."
+      "Users managed by LDAP.",
+      "User werden durch LDAP verwaltet."
   )
 
   val user_name = LocalizedString(
-    "Name",
-    "Name"
+      "Name",
+      "Name"
   )
 
   val user_permission = LocalizedString(
-    "Permission",
-    "Berechtigung"
+      "Permission",
+      "Berechtigung"
   )
 
   val user_first_login_date = LocalizedString(
-    "First login date",
-    "Datum des ersten Logins"
+      "First login date",
+      "Datum des ersten Logins"
   )
 
   val user_add = LocalizedString(
-    "Create user",
-    "User erstellen"
+      "Create user",
+      "User erstellen"
   )
 
   val user_edit = LocalizedString(
-    "Edit user",
-    "User bearbeiten"
+      "Edit user",
+      "User bearbeiten"
   )
 
   val user_update = LocalizedString(
-    "Save user",
-    "User speichern"
+      "Save user",
+      "User speichern"
   )
 
   val user_delete = LocalizedString(
-    "Delete user",
-    "User löschen"
+      "Delete user",
+      "User löschen"
+  )
+
+  val user_delete_are_you_sure = LocalizedString(
+      "Are you sure you want to delete this user?",
+      "Sind Sie sicher, dass Sie diesen Benutzer löschen möchten?"
```

Comment:
```suggestion
      "Are you sure you want to delete this user? This will not delete locations or access permissions created by this user.",
      "Sind Sie sicher, dass Sie diesen Benutzer löschen möchten? Dies löscht keine vom Benutzer erstellen Orte oder Zugangsberechtigungen."
```